### ISSUE TITLE
Homebrew for macOS distribution

### ISSUE THREAD

**Issue by adunning** [lovasoa/dezoomify-rs#79]
Rather than providing a macOS binary, could I suggest using [Homebrew](https://brew.sh) to simplify distribution on macOS? There are [lots of example Rust formulae](https://github.com/Homebrew/homebrew-core/search?q=depends_on+%22rust%22) available.

(I'll be able to create a formula myself, but won't likely have the time for a few weeks.)

**Comment by lovasoa** [comment]
Hello ! Yes, you are welcome to create and maintain a homebrew formula ! This will be appreciated. If you do, you can open a small PR to the README in this repo to link to the formula.

**Comment by lovasoa** [comment]
@adunning : Do you think you can create the formula ?

**Comment by adunning** [comment]
Yes, a few weeks was overly ambitious in the middle of term. Still on my radar!

**Comment by mildlydiverting** [comment]
(Just adding a note to say this can be closed, per #218 )
